### PR TITLE
🐛 filters out riff entries that are not saved queries

### PR DESCRIPTION
**Pull request by hlminh2000**
stop gap solution to allow dashboard SavedQueries card to identify saved queries from riff data.

### CODE REVIEW

File: src/components/UserDashboard/SavedQueries/index.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ const FileRepositoryLink = styled(Link)`
   color: ${({ theme }) => theme.primary};
 `;
 
+/**
```
Comment (joneubank) [inline src/components/UserDashboard/SavedQueries/index.js]: 👍 Thank you for comments like these.

File: src/components/UserDashboard/SavedQueries/index.js [hunk 2]
```diff
@@ -44,7 +50,8 @@ export const MySavedQueries = compose(
     api,
     theme,
   }) => {
-    const Header = <CardHeader title="Saved Queries" badge={queries.length} />;
+    const savedQueries = queries.filter(isSavedQuery);
```
Comment (joneubank) [inline src/components/UserDashboard/SavedQueries/index.js]: Should this filtering be done in the service that fetches the queries? The component doesn't want to own the logic of separating queries from v-studies

Comment (joneubank) [reply src/components/UserDashboard/SavedQueries/index.js]: Just looked at the state provider, it doesnt separate the fetching into a Riff service file. So can just move the filtering there.......

Comment (hlminh2000) [reply src/components/UserDashboard/SavedQueries/index.js]: great call! :) thanks!

**Pull request merged**